hadoop-ozone/dist/src/main/smoketest/auditparser/auditparser.robot

```diff
@@ -48,7 +50,7 @@ Testing audit parser
                        Should Contain       ${result}  ${user}
     ${result} =        Execute              ozone auditparser "${auditworkdir}/audit.db" query "select count(*) from audit where op='CREATE_VOLUME' and RESULT='SUCCESS'"
     ${result} =        Convert To Number     ${result}
-                       Should be true       ${result}>${count}
+                       Should be true       ${result}>=1
```

Comment:
This is because we are no longer creating 5 volumes right?

Reply:
Yes, `ombg` and `ockg` only create one volume.